matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerExporter.java

```diff
@@ -220,11 +221,129 @@ private static void createBuses(Network network, MatpowerModel model, Context co
         createTransformerStarBuses(network, model, context);
     }
 
+    private static boolean isEmergencyLimit(LoadingLimits.TemporaryLimit limit) {
+        return limit.getAcceptableDuration() <= 60;
+    }
+
+    private static Optional<LoadingLimits.TemporaryLimit> findShortTermLimit(Stream<LoadingLimits.TemporaryLimit> limitStream) {
+        return limitStream.filter(limit -> !isEmergencyLimit(limit))
+                .max(Comparator.comparing(LoadingLimits.TemporaryLimit::getAcceptableDuration));
+    }
+
+    private static Optional<LoadingLimits.TemporaryLimit> findEmergencyLimit(Stream<LoadingLimits.TemporaryLimit> limitStream) {
+        return limitStream.filter(MatpowerExporter::isEmergencyLimit)
+                .min(Comparator.comparing(LoadingLimits.TemporaryLimit::getAcceptableDuration));
+    }
+
+    private static Optional<LoadingLimits.TemporaryLimit> previousLimit(Collection<LoadingLimits.TemporaryLimit> limits, LoadingLimits.TemporaryLimit limit) {
+        return limits.stream().filter(l -> l.getAcceptableDuration() > limit.getAcceptableDuration())
+                .min(Comparator.comparing(LoadingLimits.TemporaryLimit::getAcceptableDuration));
+    }
+
+    private static Optional<LoadingLimits.TemporaryLimit> nextLimit(Collection<LoadingLimits.TemporaryLimit> limits, LoadingLimits.TemporaryLimit limit) {
+        return limits.stream().filter(l -> l.getAcceptableDuration() < limit.getAcceptableDuration())
+                .max(Comparator.comparing(LoadingLimits.TemporaryLimit::getAcceptableDuration));
+    }
+
+    private static double toMva(double ampere, VoltageLevel vl) {
```

Comment:
Current?

Reply:
Ok so I also renamed method to "toApparentPower"